sdcm/sct_events/loaders.py

```diff
@@ -215,15 +216,18 @@ def msgfmt(self):
                                           regex="Using optimized driver")
 CassandraStressLogEvent.add_subevent_type("SchemaDisagreement", severity=Severity.WARNING,
                                           regex="No schema agreement")
-
+CassandraStressLogEvent.add_subevent_type("AuthenticationError", severity=Severity.WARNING,
```

Comment:
Let's do the regex a bit more specific to the actual issue i.e. that it's consistency error with cl ONE

And add a comment here, with a reference to the issue